src/turbopuffer.ts

```diff
@@ -17,6 +17,13 @@ export { TurbopufferError } from "./httpClient";
 export type Id = string | number;
 export type AttributeType = null | string | number | string[] | number[];
 export type Attributes = Record<string, AttributeType>;
+export type Schema = Record<string, {
+  type?: string;
+  bm25?: boolean | Record<string, string|boolean>;
```

Comment:
Instead of `string|boolean`, we could be more explicit here about which keys with which value types to allow. Might cause some problems in the future if it gets out of sync with server-side changes. Thoughts?

Reply:
This is reasonable IMO, we need to add numbers to that union type eventually (when we expose k1 and b), before then, should be OK